analysis/export/CSVExporter/src/main/kotlin/de/maibornwolff/codecharta/exporter/csv/CSVExporter.kt

```diff
@@ -41,6 +43,7 @@ class CSVExporter() : Callable<Void>, InteractiveParser {
     override val name = NAME
     override val description = DESCRIPTION
 
+    private val logger = KotlinLogging.logger {}
     companion object {
```

Comment:
I don't like to stick it to the companion object.
Let's use breaklines if it makes sense to separate code blocks that belong together logically.